### ISSUE TITLE
Icons after log-in Adwaita, then change to Papirus

### ISSUE THREAD

**Issue by coxackie** [micheleg/dash-to-dock#640]
Not sure whether this is an issue with the extension itself or with GNOME shell. I am using the Papirus icon theme; however, quite often, upon log-in, Adwaita icons appear in the dock, and after a second or two they change (refresh) to Papirus. I am using GNOME Shell 3.26 on Arch.

**Comment by micheleg** [comment]
That's fairly strange.... Can you please confirm that nothing strange happens without the extension and that the problem happens also when disabling all but dash-to-dock  extensions?

**Comment by micheleg** [comment]
Note to myself: most likely wayland related.

**Comment by coxackie** [comment]
Indeed, it appears that when I log in using GNOME on Xorg the latency in changing from Adwaita to Papirus is not there.
